utils/generate_types/overrides-test.d.ts

```diff
@@ -312,6 +312,24 @@ declare global {
   export namespace PlaywrightTest {
     export interface Matchers<R, T = unknown> {
     }
+
+    export interface TestFixtures extends KeyValue {
+    }
+
+    export interface WorkerFixtures extends KeyValue {
+    }
+
+    export interface ConfigTestOptions extends KeyValue {
+    }
+
+    export interface ConfigWorkerOptions extends KeyValue {
+    }
```

Comment:
In an ideal world, I'd use `Global` instead of using `Config` v. `Project` prefix to imply the same thing, but we already have a convention.